src/models/storage/batch.py

```diff
@@ -41,8 +41,9 @@ def __init__(self, frames, outcomes=None, temp_outcomes=None,
             outcomes = dict()
         if temp_outcomes is None:
             temp_outcomes = dict()
+        # store the batch with columns sorted
         if isinstance(frames, DataFrame):
-            self._frames = frames
+            self._frames = frames[sorted(frames.columns)]
```

Comment:
Can any operator leave it unsorted?

Reply:
The Readers while creating a batch from video frames can leave it unsorted. Either we add this constraint while reading or I thought of adding it in the constructor itself.

Reply:
Batch(pd.DataFrame({'id': 1, 'data': 2})). The batch object is typically creating in this manner. So the order of columns is dependent on the order of keys in the dictionary. So to avoid inconsistency across the system I added the sorted requirement.

Reply:
Sounds good. We could add a sort operator later.